analytics/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from zerver.models import Realm, UserProfile
+from zerver.lib.str_utils import ModelReprMixin
+
+# only would ever touch by hand
+class Anomaly(ModelReprMixin, models.Model):
+    info = models.CharField(max_length=1000) # type: text_type
+
+    def __unicode__(self):
+        # type: () -> text_type
+        return u"<Anamoly: %s... %s>" % (self.info, self.id)
+
+class RealmCount(ModelReprMixin, models.Model):
+    realm = models.ForeignKey(Realm) # type: Realm
+    value = models.BigIntegerField() # type: int
+    property = models.CharField(max_length=40) # type: text_type
```

Comment:
Maybe have RealmCount have the fields `realm', `bucket`, and `value`?  For cosmetic reasons I would make `value` be the third field here.